.github/pull_request_template.md

```diff
@@ -2,6 +2,34 @@
 ```
 type(scope): subject
 ```
+
+Available types:
+- `feat`: A new feature
+- `fix`: A bug fix
+- `docs`: Documentation only changes
+- `style`: Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
+- `refactor`: A code change that neither fixes a bug nor adds a feature
+- `perf`: A code change that improves performance
+- `test`: Adding missing tests or correcting existing tests
+- `build`: Changes that affect the build system or external dependencies (example scopes: gulp, broccoli, npm)
+- `ci`: Changes to our CI configuration files and scripts (example scopes: Travis, Circle, BrowserStack, SauceLabs)
+- `chore`: Other changes that don't modify src or test files
+- `revert`: Reverts a previous commit
+Example:
+feat(api): add user login endpoint
+
+### Description
```

Comment:
I can see, all these (description, link of the issue, testing details) got duplicated. Do you want to remove this?